apps/extension/tsconfig.json

```diff
@@ -1,7 +1,11 @@
 {
   "extends": "plasmo/templates/tsconfig.base",
-  "exclude": ["node_modules"],
-  "include": [".plasmo/**/*", "./**/*.ts", "./**/*.tsx", "next-auth.d.ts"],
+  "include": [
+    ".plasmo/**/*",
+    "node_modules/@agreeto/auth/next-auth.d.ts",
```

Comment:
should look into publically exporting this type and use it through `compilerOptions.types` instead. Ran into some issues with that so left it like this for now